SEC/Equities/EquityInstruments.rdf

```diff
@@ -768,29 +859,37 @@
 		<skos:definition xml:lang="en">grants or bestows ownership via contract</skos:definition>
 	</owl:ObjectProperty>
 	
-	<owl:ObjectProperty rdf:about="&fibo-sec-eq-eq;hasCoupon">
+	<owl:ObjectProperty rdf:about="&fibo-sec-eq-eq;hasAdjustableDividendRate">
+		<rdfs:subPropertyOf rdf:resource="&lcc-lr;has"/>
+		<rdfs:label>has adjustable dividend rate</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-sec-eq-eq;PreferredDividend"/>
+		<rdfs:range rdf:resource="&fibo-fnd-acc-cur;PercentageMonetaryAmount"/>
+		<skos:definition>indicates the pre-arranged variable dividend rate, typically specified in the prospectus as a formula based on a benchmark, for a preferred share</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-sec-eq-eq;hasDistributionMethod">
 		<rdfs:subPropertyOf rdf:resource="&lcc-lr;has"/>
-		<rdfs:label>has coupon</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-sec-eq-eq;PreferredShare"/>
+		<rdfs:label>has distribution method</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-sec-eq-eq;Dividend"/>
+		<rdfs:range rdf:resource="&fibo-sec-eq-eq;DividendDistributionMethod"/>
+		<skos:definition>indicates the method by which dividend payments are to be distributed</skos:definition>
```

Comment:
don't need "indicates"

Reply:
The property and definition needs to start with a verb, per our policy, especially in the case of an object property.  Please suggest an alternative verb if you don't think 'indicates' is appropriate.

Reply:
I thought the key point was that the definition can be substituted into a sentence in place of the original name. Let's say we have:
 "XincPrefA hasDistributionMethod Reinvestment"
Using the current definition gives:
 "XincPrefA indicates the method by which dividend payments are to be distributed Reinvestment" 
which does not work at all IMO
So better would be:
 "XincPrefA has method by which dividend payments are to be distributed of Reinvestment"